adapters/huaweiads/huaweiadstest/supplemental/bad_resquest_no_support_audio.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Nitpick: typo in filename. Can you change `bad_resquest_` to `bad_request_`?

Reply:
done, thank you